pyxem/utils/vector_utils.py

```diff
@@ -61,6 +61,54 @@ def detector_to_fourier(k_xy, wavelength, camera_length):
     return k
 
 
+def detector_px_to_3D_kspace(peak_coord, beam_wavelen, det2sample_len, pixel_len):
```

Comment:
I am fairly certain that this reproduces `detector_to_fourier(k_xy, wavelength, camera_length)` in `vector_utils`, albeit with some extra pixel-calibrated transformations. I would prefer to see that imported and used.

Reply:
`detector_to_fourier` does the same transformation, but assumes coordinates to be calibrated in reciprocal Ångström. This is done by setting the diffraction calibration on the signal, which is probably a better place to store the calibration. The difference is that `detector_px_to_3D_kspace` takes into account curvature of the Ewald sphere when calculating the x and y coordinates. I would be interested in a comparison of how large the error is for typical electron microscope wavelengths.